src/validators/BaseValidator.ts

```diff
@@ -76,18 +77,18 @@ export abstract class BaseValidator<T> {
 	 * Sets if the validator should also run constraints or just do basic checks.
 	 * @param isValidationEnabled Whether this validator should be enabled or disabled. Set to `null` to go off of the global configuration.
 	 */
-	public setValidationEnabled(isValidationEnabled: boolean | null): this {
+	public setValidationEnabled(isValidationEnabled: boolean | (() => boolean) | null): this {
 		const clone = this.clone();
 		clone.isValidationEnabled = isValidationEnabled;
 		return clone;
```

Comment:
I would also add onto the doc comment that this accepts a function that returns a boolean.

Reply:
🖕

Reply:
👆this is the righ emoji